ci-operator/step-registry/cucushift/logging/cucushift-logging-ref.yaml

```diff
@@ -0,0 +1,30 @@
+ref:
+  as: cucushift-logging
```

Comment:
I think no need to create a new step, the `cucushift-e2e` can run logging tests. You should be able to do that with below configs in `xxx_multi-nightly-openshift-logging-5.8.yaml`
```
E2E_RUN_TAGS: '@logging5.8'
E2E_TEST_TYPE: "destructive"
```

Reply:
I think that's because logging test cases in cucushift only update logging versions, but OCP versions are not updated. E.g., https://github.com/openshift/verification-tests/blob/master/features/logging/cluster_log_forwarder.feature#L141-L142

Reply:
Yes, if we update OCP version, all logging cases will be selected by OCP CI, this is not what we want.

Reply:
OK